### DISCUSSION TITLE
Collaborators' Discussion

### DISCUSSION THREAD

**Discussion by Zo-Bro-23** [anuraghazra/github-readme-stats#2443] [General]
Talk about:
- Pull requests
- New features (enhancement)
- Solutions for bugs (issues)

...and more!

This is primarily meant for active collaborators or those looking to collaborate. It contains the issues/PRs we deem important. To see the issues/PRs the community deems important, go to https://github.com/anuraghazra/github-readme-stats/issues/1935. Feature requests and bug reports are best if submitted as an issue.

**Comment by Zo-Bro-23** [comment]
# GRS Action
Create a GitHub Action for users to add GRS cards to their profiles. Might be easier once #2409 is merged.

See (#2179) for more.

**Reply by Zo-Bro-23** [reply]
GRS Action will be easier for users to set up than a Vercel deployment.

See [this](https://github.com/anuraghazra/github-readme-stats/issues/2415#issuecomment-1399227179).

**Reply by Zo-Bro-23** [reply]
Switching to an action still doesn't solve ***all*** problems, however, since we still need to be able to cope with the load from existing users (who will continue using the API even if update the README to recommend the Action instead). Just something to keep in mind.

**Reply by Zo-Bro-23** [reply]
#2473 publishes GRS as an NPM module, making it easier to work on the Action.

**Reply by Zo-Bro-23** [reply]
#2537 adds a GRS GitHub Action.

**Comment by Zo-Bro-23** [comment]
# Ranking System
- Change the ranking system
- Add text descriptions for rank on hover (percentage of top users)

Waiting for #1633.

See (#2265) for more.

**Comment by Zo-Bro-23** [comment]
# Custom Cards
#2409 waiting for review and merge. Will make many features easier to add (#1563, #2179, and more).

See (#2409) for more.

**Comment by Zo-Bro-23** [comment]
# Vercel maxDuration fix
#2424 waiting for review and merge. Will make it easier for users to deploy their own instance on Vercel.

See (#2424) for more.

**Comment by Zo-Bro-23** [comment]
# GitHub Rate Limiting
GitHub limits their GraphQL API to 5000 per hour, which causes issues with GRS.

**Reply by Zo-Bro-23** [reply]
# Automatic Workflow for Issue Handling
#2427 proposes GH Actions workflow for automatically closing issues related to this (and pointing them all to a single issue).

**Reply by Zo-Bro-23** [reply]
# Multiple PATs/Instances
Multiple PATs and proxying to multiple Vercel instances can be considered.

**Reply by Zo-Bro-23** [reply]
# Community Instances
Adding community instances to the README should also be considered.

**Reply by rickstaa** [reply]
# Add `stale-if-error` header

By adding the `stale-if-error` heading, we might have the error show less frequency to users (see https://github.com/anuraghazra/github-readme-stats/pull/2448.

**Comment by rickstaa** [comment]


**Reply by Zo-Bro-23** [reply]
Sure!

**Comment by rickstaa** [comment]
# Organization stats

One of our most requested features is allowing people to include organization stats. I created https://github.com/anuraghazra/github-readme-stats/pull/2459 based on https://github.com/anuraghazra/github-readme-stats/pull/1122. @anuraghazra and @Zo-Bro-23 let me know what you think. I will add tests and documentation if you agree that we can add this under an experimental flag.

**Reply by Zo-Bro-23** [reply]
Adding under an experimental flag seems okay. We'll see how well it works and how it affects the accuracy of other stats before we decide whether to add it to the public instance or not.

**Comment by Zo-Bro-23** [comment]
# Removing `count_private` option
See #2504 for more.

**Comment by rickstaa** [comment]
# Add pie/Doughnut chart

See https://github.com/anuraghazra/github-readme-stats/issues/1650.

**Comment by rickstaa** [comment]
# Setup Crowdin integration

See https://github.com/anuraghazra/github-readme-stats/pull/2489.
